hazelcast/src/main/java/com/hazelcast/wan/impl/DelegatingWanReplicationScheme.java

```diff
@@ -101,21 +102,31 @@ public void publishReplicationEventBackup(WanReplicationEvent event) {
     /**
      * Publishes a replication event to all publishers to which this publisher
      * delegates.
+     * Silently skips publishers not supporting republication.
      */
     public void republishReplicationEvent(WanReplicationEvent wanReplicationEvent) {
```

Comment:
not used method?

Reply:
also `addPublisher` is not used in this class?

Reply:
+ same for `destroyMapData`

Reply:
if these are overridden in EE, maybe we can put a short comment for that.

Reply:
All of these are used from EE. I might separate them into an OS and EE `DelegatingWanReplicationScheme`/`EnterpriseDelegatingWanReplicationScheme` but with the previous WAN SPI, I've started not liking separate classes for OS and EE too much :)

I'll add the comment then.